packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,280 +48,79 @@ export default {
-                title: 'true'
-              },
-              {
-                value:
-                  '<sd-radio value="1">Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'false'
-              }
-            ]
-          }
-        ],
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group name="radio-group" size="lg">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+
+      <sd-radio-group name="radio-group" size="sm">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
- * Use the `invalid` attribute to mark the element is not in a format or a value the application will accept.
+ * Use the `orientation` attribute to set the orientation of the radio buttons.
  */
 
-export const Invalid = {
-  parameters: { controls: { exclude: ['size', 'value', 'required', 'invalid'] } },
-  render: (args: any) => {
-    return html`<form>
-      ${generateTemplate({
-        constants: [
-          { type: 'attribute', name: 'required', value: true },
-          { type: 'attribute', name: 'value', value: '' }
-        ],
-        args
-      })}
-      <sd-button style="margin-top: 16px" type="submit">Submit</sd-button>
-    </form>`;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('sd-button');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const Orientation = {
+  name: 'Orientation',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group name="radio-group" orientation="vertical">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Vertical Group</label>
+      </sd-radio-group>
+
+      <sd-radio-group name="radio-group" orientation="horizontal">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Horizontal Group</label>
+      </sd-radio-group>
+    </div>
+  `
 };
 
 /**
  * Use the `required` attribute to mark the element as required. This can be used for form validation purposes.
+ *
+ *
+ * **Accessibility:** To ensure screen-reader compatibility, consider including a statement such as "Fields marked with an asterisk (*) are required" at the start of the form.
```

Comment:
I guess this is only Code relevant?
@coraliefeil @MartaPintoTeixeira will you add this to figma and mark it as code-only (or dev/storybook - don't know the exact wording)?

Reply:
We have this info in the accessibility section here in [Figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2213-20986&t=AcHb8BRC0LXsi0AC-11).

We just gathered all a11y info for each comp in this section. We need to rework that at a later point ...